src/controllers/permissions.js

```diff
@@ -0,0 +1,64 @@
+import { checkIfExistsInDb } from '../utils/searchDb';
+import { Resource, Permission } from '../models';
+/**
+ * @class
+ * @description Class based Permissions Controller
+ */
+class Permissions {
+  /**
+   * @description Sets the permission for a given role to a particular resource
+   * @static
+   * @param {object} req object
+   * @param {object} res object
+   * @param {method} next method
+   * @returns {object } Updated Role information
+   * @memberof Permissions
+   * @type {object}
+   */
+  static async setPermissions(req, res, next) {
+    let updatedPermissions;
+    const { roleId } = req.params;
+    const { resourceId, ...permissions } = req.body;
+    try {
+      // Check if Resource Exists
```

Comment:
Remove comment

Reply:
The comment is to explain the flow of the code.